src/rust/ide/src/view/project.rs

```diff
@@ -91,20 +110,44 @@ pub async fn recreate_if_missing(project:&model::Project, path:&FilePath, defaul
     Ok(())
 }
 
+/// Add main method definition to the given module, if the method is not already defined.
+///
+/// The lookup will be done using the given `main_ptr` value.
+pub fn add_main_if_missing
+(module:&model::Module, main_ptr:&MethodPointer, parser:&Parser) -> FallibleResult<()> {
+    if module.lookup_method(main_ptr).is_err() {
+        let mut info  = module.info();
+        let main_code = default_main_method_code();
+        let main_ast  = parser.parse_line(main_code)?;
+        info.add_ast(main_ast,double_representation::module::Placement::End)?;
+        module.update_ast(info.ast);
+
+        // TODO [mwu] Add imports required by the default main definition (when needed).
```

Comment:
Let's add it right now, or put here a reference to issue with managing new import syntax.